booking/storage.py

```diff
@@ -0,0 +1,57 @@
+from flask import app
+
+
+class StorageError(Exception):
+    pass
+
+
+class TicketStorage:
+
```

Comment:
Consider using Docstrings in which you can explain briefly what each class is responsible for.

Reply:
This class is responsible for adding and getting events and also we keep in this storage class orders, so if this is only prototype then we can keep all this information in one class but the name should be Storage, not TicketStorage.